packages/projection-typeorm/src/createDataSource.ts

```diff
@@ -71,6 +71,9 @@ const namingOverrides: Partial<NamingStrategyInterface> = {
   },
   tableName(targetName, userSpecifiedName) {
     return userSpecifiedName || toTableName(snakeCase(targetName));
+  },
+  uniqueConstraintName(tableName, columnNames) {
+    return `UQ_${toTableName(tableName)}_${columnNames.join('_')}}`;
```

Comment:
Could `UC` be the right prefix for unique constraints?

Reply:
Sure if it's the more common convention. TypeORM used UQ by default so I left that, just replaced the hex hash with readable names

Reply:
Nice to learn about the TypeORM default. Thank you